src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -1668,14 +1668,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return node.IsKind(SyntaxKind.NotExpression)
         End Function
 
-        Public Function IsConditionalAnd(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsConditionalAnd
-            Return node.Kind() = SyntaxKind.AndAlsoExpression
-        End Function
-
-        Public Function IsConditionalOr(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsConditionalOr
-            Return node.Kind() = SyntaxKind.OrElseExpression
-        End Function
```

Comment:
Duplicated of helpers defined a few lines above at https://github.com/dotnet/roslyn/pull/41525/files#diff-c02ddd970a39731d626b502f7a743093R1659